packages/typespec-go/src/emitter.ts

```diff
@@ -15,22 +15,13 @@ import { generatePolymorphicHelpers } from '../../codegen.go/src/polymorphics.js
 import { generateResponses } from '../../codegen.go/src/responses.js';
 import { generateTimeHelpers } from '../../codegen.go/src/time.js';
 import { generateServers } from '../../codegen.go/src/fake/servers.js';
-import { generateServerFactory } from '../../codegen.go/src/fake/factory.js';
+//import { generateServerFactory } from '../../codegen.go/src/fake/factory.js';
```

Comment:
Remove the commented code

Reply:
I left this and the point of invocation as we still need to decide what we'll do with the fake server factory (will update comment with TODO).